### PR TITLE
[post] Making Nette and Typo3 to community packages

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
@ulco Hi Michal, 
I've put down few notes before next release, so people know what is changing and why it's better :)

Could you check it and share your thoughts, what would you improve?
There are few bugfixes since last release, wo we plan to release this post along new Rector version next week.

Thanks

File: data/blog/posts/2022/2022-08-05-separating-typo3-and-nette-as-community-packages.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "Separating Typo3 and Nette as Community Packages"
+perex: |
+    When Rector started, it was a small project that handled upgrading a vast amount of PHP packages.
+    <br><br>
+    As the project grew and expanded, more local PHP communities joined with community packages that build custom rules on top Rector core.
+    <br><br>
+    It makes sense to separate these projects from the core and let the community handle them. Who does a better job at growing the vegetable than farmers themselves, right?
+---
+
+To this day, there are around 10 known community packages that are [maintained by the community](https://github.com/rectorphp/rector#empowered-by-rector-community-heart). These include [CraftCMS](https://github.com/craftcms/rector), [Shopware](https://github.com/FriendsOfShopware/shopware-rector) or famous [Drupal Rector bot](https://www.drupal.org/blog/accelerating-drupal-9-module-and-theme-readiness-with-automated-patches) that automates the upgrades for Drupal 9.
```
Comment (samsonasik) [inline data/blog/posts/2022/2022-08-05-separating-typo3-and-nette-as-community-packages.md:12]: 9 based on the list: 

```suggestion
To this day, there are around 9 known community packages that are [maintained by the community](https://github.com/rectorphp/rector#empowered-by-rector-community-heart). These include [CraftCMS](https://github.com/craftcms/rector), [Shopware](https://github.com/FriendsOfShopware/shopware-rector) or famous [Drupal Rector bot](https://www.drupal.org/blog/accelerating-drupal-9-module-and-theme-readiness-with-automated-patches) that automates the upgrades for Drupal 9.
```

**Comment by TomasVotruba** [comment]
@lulco Hi Michal, 
I've put down few notes before next release, so people know what is changing and why it's better :)

Could you check it and share your thoughts, what would you improve?
There are few bugfixes since last release, wo we plan to release this post along new Rector version next week.
If you want rename namespace in the rector-nette and tag it by then, this post would make more sense and allow easy switch.

Thanks

**Comment by lulco** [comment]
@TomasVotruba I'm OK with it ;)

**Comment by TomasVotruba** [comment]
@lulco Thanks 🙏

File: data/blog/posts/2022/2022-08-05-separating-typo3-and-nette-as-community-packages.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+That's why we believe the community package should be in the hands of people who use the framework daily. A new framework version brings new features every year, and the person to convert them to Rector rules is a passionate developer with a taste for innovation and bleeding edge.
+
+<br>
+
+That's why we decided to separate typo3 and Nette from the core and let their active communities take over. They're not part of `rector/rector` anymore, but they're Rector extensions that you can install yourself if you need those:
+
+You can find those at:
+
+* https://github.com/sabbelasichon/typo3-rector
```
Comment (samsonasik) [inline data/blog/posts/2022/2022-08-05-separating-typo3-and-nette-as-community-packages.md:34]: New rector-nette need to be listed here

Comment (TomasVotruba) [reply data/blog/posts/2022/2022-08-05-separating-typo3-and-nette-as-community-packages.md:34]: Done, thanks :+1:

**Comment by TomasVotruba** [comment]
Let's :ship: it :)

**Pull request merged**